anghammarad/src/test/scala/com/gu/anghammarad/models/SerializationTest.scala

```diff
@@ -1,29 +1,69 @@
 package com.gu.anghammarad.models
 
+import com.amazonaws.services.lambda.runtime.events.SNSEvent
+import com.amazonaws.services.lambda.runtime.events.SNSEvent.SNSRecord
 import io.circe.Json
 import io.circe.parser._
 import org.scalatest.{EitherValues, FreeSpec, Matchers}
 
+import scala.collection.JavaConverters._
 import scala.io.Source
 
 
 class SerializationTest extends FreeSpec with Matchers with EitherValues {
 
-  "parseNotification" - {
+  "Notification Serialization" - {
     val validJsonString = Source.fromURL(getClass.getResource("/notification.json")).mkString
-    val testJson = parse(validJsonString).right.value
-
-    "will parse valid json into a complete notification" in {
-      val expectedResult = Notification(
-        "Terry Pratchett",
-        Email,
-        List(Stack("postal-service"), App("clacks-overhead")),
-        "GNU Terry Pratchett",
-        "Words are important. And when there is a critical mass of them, they change the nature of the universe.",
-        List(Action("keep that name moving in the Overhead", "http://www.gnuterrypratchett.com/"))
-      )
+    val validJson = parse(validJsonString).right.value
+
+    val expectedResult = Notification(
+      "Terry Pratchett",
+      Email,
+      List(Stack("postal-service"), App("clacks-overhead")),
+      "GNU Terry Pratchett",
+      "Words are important. And when there is a critical mass of them, they change the nature of the universe.",
+      List(Action("keep that name moving in the Overhead", "http://www.gnuterrypratchett.com/"))
+    )
+
+    "parseNotification" - {
+      val testSNS = new SNSEvent.SNS()
+      val testRecord = new SNSRecord()
+      val testEvent = new SNSEvent()
+
+      testSNS.setSubject("GNU Terry Pratchett")
+      testSNS.setMessage(validJsonString)
+      testRecord.setSns(testSNS)
+
+      "will parse a SNSEvent into a complete notification" in {
+        val input = testEvent
+          input.setRecords(List(testRecord).asJava)
+        Serialization.parseNotification(testEvent).get shouldEqual expectedResult
```

Comment:
You could use our `TryValues` from `testutils` instead of `get` so we get a nice test failed message? This is fine as well though.